Modix.Services/Configuration/DiscordBotConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Modix.Services.Configuration
+{
+    public class DiscordBotConfiguration
+    {
+        public bool PurgeInvites => InvitePurging != null;
+
+        public InvitePurgingConfiguration InvitePurging { get; set; }
+
+        public class InvitePurgingConfiguration
+        {
+            public List<ulong> ExemptRoles { get; set; }
+            public ulong LoggingChannelId { get; set; }
```

Comment:
We should reuse [`ModixConfig.ModixPrivateLogChannelId`](https://github.com/discord-csharp/MODiX/blob/ec29a4557fd38c2dc3d7ecfde69221ca4fd3df7d/Modix.Data/Models/ModixConfig.cs#L25) for this.

Reply:
Good point, I could change it - I was thinking we keep the same visibility as rowboat currently has with #debug-log, as Operators don't have access to that. @Cisien / @mastorm what do you think where this should log to?

Reply:
I both like and dislike separate channels for logging

on one hand, i like seeing invites (or anything) getting censored so action can be taken
on the other, we have something like 3 or 4 logging channels.

I think somewhere in the middle, if we can reduce the number of logging channels while keeping censored actions split up, i'd be for it.